chips/cc26x2/src/ccfg.rs

```diff
@@ -99,7 +99,7 @@ impl Registers {
 }
 
 register_bitfields![
-    u8,
+    u32,
```

Comment:
How is this change related to updating rust nightly?

Reply:
So the previous type was incorrect, right? As I understand it, there are offsets greater than 8 so the associated fields wouldn't be representable in a `u8`, right?

If that's the case, is there a way/plan to add unit tests to the `register_bitfields` to check that e.g. offsets fit in the underlying integer type, that they don't overlap with one another, etc?

Reply:
Right, as far as I can tell `u8` is wrong. As for additional tests/checks...I'm not sure.